test/rubygems/test_gem_specification.rb

```diff
@@ -804,8 +804,7 @@ def test_self_load_tainted
       end
 
       full_path.taint
-      loader = Thread.new { $SAFE = 1
- Gem::Specification.load full_path }
+      loader = Thread.new { $SAFE = 1; Gem::Specification.load full_path }
```

Comment:
A glitch in the system 😆. Hopefully not a bug in `util/rubocop -A --only Style/BarePercentLiterals` 😅. Luckily it got corrected back in a later commit :)